webapp/channels/src/i18n/en.json

```
@@ -3375,7 +3375,7 @@
   "channel_loader.postedImage": " posted an image",
   "channel_loader.socketError": "Please check connection, Mattermost unreachable. If issue persists, ask administrator to [check WebSocket port](!https://docs.mattermost.com/install/troubleshooting.html#please-check-connection-mattermost-unreachable-if-issue-persists-ask-administrator-to-check-websocket-port).",
   "channel_loader.someone": "Someone",
-  "channel_loader.something": " did something new",
+  "channel_loader.something": " posted a message",
```

```suggestion
  "channel_loader.posted": " posted a message",
```
by changing the key, we give more context to the translators on what's expected

---

Good suggestion, thanks @Willyfrog! Updated to rename the existing `channel_loader.posted` to be `channel_loader.title` and then use this key as such.